src/devices/lacrosse_TX141TH_Bv2.c

```diff
@@ -0,0 +1,230 @@
+ *
+ * 0x87 0x02 0x3c 0x3b 0xe1
+ * 0x87 0x02 0x7d 0x37 0xe1
+ *
+ * 0x87 0x01 0xc3 0x31 0xd8
+ * 0x87 0x02 0x28 0x37 0xd8
+ *
+ * Developer's comment 1: because of our choice of the OOK_PULSE_PWM_TERNARY type, the input
+ * array of bits will look like this:
+ * bitbuffer:: Number of rows: 25
+ *  [00] {0} :
+ *  [01] {0} :
+ *  [02] {0} :
+ *  [03] {0} :
+ *  [04] {40} 87 02 67 39 f6
+ *  [05] {0} :
+ *  [06] {0} :
+ *  [07] {0} :
+ *  [08] {40} 87 02 67 39 f6
+ *  [09] {0} :
+ *  [10] {0} :
+ *  [11] {0} :
+ *  [12] {40} 87 02 67 39 f6
+ *  [13] {0} :
+ *  [14] {0} :
+ *  [15] {0} :
+ *  [16] {40} 87 02 67 39 f6
+ *  [17] {0} :
+ *  [18] {0} :
+ *  [19] {0} :
+ *  [20] {40} 87 02 67 39 f6
+ *  [21] {0} :
+ *  [22] {0} :
+ *  [23] {0} :
+ *  [24] {280} 87 02 67 39 f6 87 02 67 39 f6 87 02 67 39 f6 87 02 67 39 f6 87 02 67 39 f6 87 02 67 39 f6 87 02 67 39 f6
+ * which is a direct consequence of two factors: (1) pulse_demod_pwm_ternary() always assuming
+ * only one startbit, and (2) bitbuffer_add_row() not adding rows beyond BITBUF_ROWS. This is
+ * OK because the data is clearly processable and the unique pattern minimizes the chance of
+ * confusion with other sensors, particularly Bresser 3CH.
+ *
+ * Developer's comment 2: with unknown CRC (see above) the obvious way of checking the data
+ * integrity is making use of the 12 packet repetition. In principle, transmission errors are
+ * be relatively rare, thus the most frequent packet (statistical mode) should represent
+ * the true data. Therefore, in the fisrt part of the callback routine the mode is determined
+ * for the first 4 bytes of the data compressed into a single 32-bit integer. Since the packet
+ * count is small, no sophisticated mode algorithm is necessary; a simple array of <data,count>
+ * structures is sufficient. The added bonus is that relative count enables us to determine
+ * the quality of radio transmission.
+ *
+ * Copyright (C) 2017 Robert Fraczkiewicz   (aromring@gmail.com)
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ */
+#include "data.h"
+#include "rtl_433.h"
+#include "util.h"
+
+#define LACROSSE_TX141TH_BITLEN 40
+#define LACROSSE_TX141TH_BYTELEN 5  // = LACROSSE_TX141TH_BITLEN / 8
+#define LACROSSE_TX141TH_PACKETCOUNT 12
+
+typedef struct {
+    int32_t	data;	// First 4 data bytes compressed into 32-bit integer
+    int8_t count;	// Count
+} data_and_count;
+
+static int lacrosse_tx141th_bv2_callback(bitbuffer_t *bitbuffer) {
+	bitrow_t *bb = bitbuffer->bb;
+	data_t *data;
+	char time_str[LOCAL_TIME_BUFLEN];
+    local_time_str(0, time_str);
+	int i,j,k,nbytes,npacket,kmax;
+    uint8_t id=0,status=0,battery_low=0,test=0,humidity=0,maxcount;
+    uint16_t temp_raw=0;
+    float temp_c=0.0;
+    data_and_count dnc[LACROSSE_TX141TH_PACKETCOUNT];
```

Comment:
data_and_count dnc[LACROSSE_TX141TH_PACKETCOUNT] = {0}; should remove the need for a memset.